pkg/minikube/registry/drvs/hyperv/hyperv.go

```diff
@@ -105,5 +105,32 @@ func status() registry.State {
 		return registry.State{Installed: false, Running: false, Error: errorMessage, Fix: fixMessage, Doc: docURL}
 	}
 
+	// Ensure user is either a Windows Administrator or a Hyper-V Administrator.
+	adminCheckCmd := exec.CommandContext(ctx, path, `@([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole([Security.Principal.WindowsBuiltInRole] "Administrator")`)
+	adminCheckOut, adminCheckErr := adminCheckCmd.CombinedOutput()
+
+	if adminCheckErr != nil {
+		errorMessage := fmt.Errorf("%s returned %q", strings.Join(adminCheckCmd.Args, " "), adminCheckOut)
+		fixMessage := "Unable to determine current user's administrator privileges"
+		return registry.State{Installed: false, Running: false, Error: errorMessage, Fix: fixMessage, Doc: docURL}
+	}
+
+	hypervAdminCheckCmd := exec.CommandContext(ctx, path, `@([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole(([System.Security.Principal.SecurityIdentifier]::new("S-1-5-32-578")))`)
+	hypervAdminCheckOut, hypervAdminCheckErr := hypervAdminCheckCmd.CombinedOutput()
+
+	if hypervAdminCheckErr != nil {
+		errorMessage := fmt.Errorf("%s returned %q", strings.Join(hypervAdminCheckCmd.Args, " "), hypervAdminCheckOut)
+		fixMessage := "Unable to determine current user's Hyper-V administrator privileges."
+		return registry.State{Installed: false, Running: false, Error: errorMessage, Fix: fixMessage, Doc: docURL}
```

Comment:
please ensure that if we provide a dockURL there is something on hyperv docs that helps ppl on this.
if not no link should be given